bin/ledger2beancount-commodities

```diff
@@ -10,19 +10,20 @@
 use strict;
 use warnings;
 
-my $commodities_date = "1970-01-01";  # unix epoch, just because
-my $indent = "  ";
+use YAML::XS qw(LoadFile);
+
+my $config = LoadFile('ledger2beancount.yml');
```

Comment:
it would be better to do the usual dance of searching the conf file in multiple places here, with fallbacks. E.g., look first in current dir (as you do here), and the in the XDG dir ~/.config/ . There is probably some Perl machinery to do that automatically (?)